lib/view/report.dart

```diff
@@ -417,7 +417,7 @@ class _ReportPage extends State<ReportPage> with SingleTickerProviderStateMixin
                                   Text(
                                     "各地最大震度",
                                     style: TextStyle(
-                                      color: context.colors.onSurface,
+                                      color: CupertinoColors.secondaryLabel.resolveFrom(context),
```

Comment:
```suggestion
                                      color: CupertinoColors.label.resolveFrom(context),
```
把這個改成亮一點的 label 感覺會比較好